### DISCUSSION TITLE
What is causing this when I try to clone a voice and use it?

### DISCUSSION THREAD

**Discussion by Barabbasbarabbas** [rsxdalv/TTS-WebUI#144] [Q&A]
F:\one-click-installers-tts-6.0\installer_files\env\lib\site-packages\gradio\processing_utils.py:176: UserWarning: Trying to convert audio automatically from int32 to 16-bit int format.
  warnings.warn(warning.format(data.dtype))

**Comment by rsxdalv** [comment]
It doesn't affect the output. It's because gradio converts it automatically, but then complains about it. If we converted the audio on our side, gradio wouldn't complain, but it's extra work.
